### PR TITLE
`Some` function to check if tuple contains value

### PR DESCRIPTION
**Pull request by nickisyourfan**
Issue #126 

Provides built in function `some` to provide support for checking if tuple contains value.

### CODE REVIEW

File: src/function/builtin.rs [hunk 1]
```diff
@@ -148,6 +148,25 @@ pub fn builtin_function(identifier: &str) -> Option<Function> {
...
+                        }
+                    }
+                } else {
+                    if a.contains(&b) {
+                        return Ok(Value::Boolean(true));
+                    }
+                }
+                Ok(Value::Boolean(false))
+            } else {
+                Ok(Value::Boolean(false))
```
Comment (ISibboI) [inline src/function/builtin.rs:167]: This should be an error. If the first argument is wrong, then `Err(EvalexprError::expected_tuple(wrong_first_value))`, and if the second value is wrong (in the `contains_any` function which expects a tuple as second argument), then `Err(EvalexprError::expected_tuple(wrong_second_value))`.

File: src/function/builtin.rs [hunk 2]
```diff
@@ -148,6 +148,25 @@ pub fn builtin_function(identifier: &str) -> Option<Function> {
             let result_index = if arguments[0].as_boolean()? { 1 } else { 2 };
             Ok(arguments.swap_remove(result_index))
         })),
+        "some" => Some(Function::new(move |argument| {
+            let arguments = argument.as_tuple()?;
+            if let (Value::Tuple(a), b) = (&arguments[0].clone(), &arguments[1].clone()) {
```
Comment (ISibboI) [inline src/function/builtin.rs:153]: Using the bracket access operator may panic. I would first check if the number of arguments is exactly two. You can use `argument.as_fixed_len_tuple(2)?` in the previous line to do that automatically.

File: src/function/builtin.rs (hunk 1 above)
Comment (nickisyourfan) [reply src/function/builtin.rs]: Error handling updated! Please let me know if this new handling is appropriately and I can resolve the comment.

File: src/function/builtin.rs (hunk 2 above)
Comment (nickisyourfan) [reply src/function/builtin.rs:173]: Thanks! Applied. If no further detail, please let me know and I can resolve.

**Comment by nickisyourfan** [comment]
@ISibboI -- Added tests and updated docs as well. Thanks for your reviews and input!

**Review by ISibboI — APPROVED** [review]
Thanks, looks great!

**Comment by ISibboI** [comment]
I will fix the remaining minor issues myself. Thank you a lot for your contribution!

**Pull request merged**